configs/next/packages/libnxz/stage_1

```diff
@@ -0,0 +1,106 @@
+
+# POWER NX zlib compliant library build parameters for stage 1  64 bits
+# ======================================================================
+#
+
+# Tell the build system to hold the temp install folder
+ATCFG_HOLD_TEMP_INSTALL='no'
+# Tell the build system to hold the temp build folder
+ATCFG_HOLD_TEMP_BUILD='no'
+# Build in a new directory
+ATCFG_BUILD_STAGE_T='dir'
+
+atcfg_pre_configure() {
+	# Prepare configure_build and configure_host variables, which are
+	# passed as parameters to configure as --build and --host,
+	# respectively.
+	#
+	# The variable configure_target and the parameter --target are not
+	# required because this is not a Canadian Cross Compiler.
+	#
+	# The variable host is passed down to this file and holds the system
+	# where advance toolchain is being built on.
+	#
+	# Native builds may have two targets, i.e.: 32 and 64 bits, regardless
+	# of the word size of the building system. This could trick the
+	# configure script into thinking that this is a cross build, when it is
+	# not, e.g.:
+	#   configure --build=powerpc64-linux-gnu --host=powerpc-linux-gnu
+	#
+	# Therefore, on native builds, we force configure_build to have the
+	# same value as configure_host.
+	configure_host=$(find_build_target ${AT_BIT_SIZE})
+	if [[ "${cross_build}" == 'no' ]]; then
+		configure_build=${configure_host}
+	else
+		configure_build=${host}
+	fi
+
+	# Prepare the directory variables: configure_prefix, configure_bindir,
+	# configure_libdir, and configure_includedir, which are passed as
+	# arguments to configure as --prefix, --bindir, --libdir, and
+	# --includedir, respectively.
+	#
+	# On native builds, files are installed directly under the main prefix,
+	# whereas on cross builds, a '/usr' suffix is added to it.
+	if [[ "${cross_build}" == 'yes' ]]; then
+		configure_prefix="${dest_cross}/usr"
+	else
+		configure_prefix="${at_dest}"
+	fi
+	# The bin and lib directories depend on the word size, e.g.
+	# configure_bindir may be set to bin, bin32, or bin64.
+	configure_libdir=${configure_prefix}/$(find_build_libdir ${AT_BIT_SIZE})
+}
+
+atcfg_configure() {
+	# libnxz do not use autoconf
+	${SUB_MAKE} clean -C ${ATSRC_PACKAGE_WORK}
+	true
+}
+
+atcfg_make() {
+	# In gcc10 the default is now -fno-common. In order to not have multiple
+	# definitions -fcommon is needed.
+	configure_cc="${at_dest}/bin/${target64:-${target}}-gcc"
+	PATH="${at_dest}/bin:${PATH}" \
+	${SUB_MAKE} -C ${ATSRC_PACKAGE_WORK} \
+	OPTCC="${configure_cc} -m${AT_BIT_SIZE}" \
+	FLG="-fcommon"
+}
+
+atcfg_make_check() {
+	# Package testing is not done on a cross build.
+	if [[ "${cross_build}" == 'no' ]]; then
+		PATH=${at_dest}/bin:${PATH} ${SUB_MAKE} check -C ${ATSRC_PACKAGE_WORK}
+	fi
+}
+
+atcfg_pre_install() {
+	mkdir -p ${install_transfer}/lib64/
+}
+
+atcfg_install() {
+	# libnxz does not have a make install
+	PATH=${at_dest}/bin:${PATH} \
+		install -m 755 ${ATSRC_PACKAGE_WORK}/libnxz.so ${install_transfer}/lib64/libnxz.so
```

Comment:
Likewise for `configure_libdir`.

This is also missing `libnxz.a` and probably a header.  Need to confirm.

Notice that newer versions (still in develop branch) will require changes to this line.